Content.Shared/Radio/RadioChannelPrototype.cs

```diff
@@ -0,0 +1,16 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Radio
+{
+    [Prototype("radioChannel")]
+    public sealed class RadioChannelPrototype : IPrototype
+    {
+        [ViewVariables] [DataField("name")] public string Name { get; private set; } = string.Empty;
+        [ViewVariables] [DataField("keycode")] public char KeyCode { get; private set; } = '\0';
+        [ViewVariables] [DataField("channel")] public int Channel { get; private set; } = 0;
```

Comment:
Comments, e.g. "KeyCode is the prefix used to determine what channel a message corresponds to).